packages/engine/src/scene/components/LinkComponent.ts

```diff
@@ -121,6 +182,12 @@ export const LinkComponent = defineComponent({
     useEffect(() => {
       setComponent(entity, BoundingBoxComponent)
       setComponent(entity, InputComponent, { highlight: true, grow: true })
+      if (!getState(EngineState).isEditor) {
```

Comment:
Is there a good reason not to make this reactive?

Reply:
Does isEditor change if you click the play button in the editor? If it  does then I should make it reactive and change the other one as well.

Reply:
It does not currently

Reply:
> It does not currently

Should I make it do so? I worry about how many possible fixes it might entail. Might warrant its own Issue?

Reply:
i dont think it's worth worrying about just yet, we wont be tackling this for a little while